modules/api/php/views/visit/recording/metadata/specific.class.inc

```diff
@@ -0,0 +1,98 @@
+<?php declare(strict_types=1);
+/**
+ * PHP Version 7
+ *
+ * @category ApiViews
+ * @package  LORIS
+ * @author   Cecile Madjar <cecile.madjar@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\api\Views\Visit\Recording\Metadata;
+
+/**
+ * Creates a representation of an image specific header format following the api
+ * response specifications.
+ *
+ * @category ApiViews
+ * @package  LORIS
+ * @author   Cecile Madjar <cecile.madjar@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Specific
```

Comment:
This is a weird class name. It can be something more specific like "SpecificHeader"

Reply:
this is the exact same behaviour as the image endpoints:
- for image: `..../headers/specific` and `..../headers/full` (and `..../headers/summary`)
- for recording: `.../metadata/specific` and `..../metadata/full`

I figured that for the user, it would be best to keep the same endpoint to the difference that headers is being replaced by metadata for recordings as headers does not really make sense for a recording